styleguide/checkstyle.xml

```diff
@@ -145,27 +145,32 @@ module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
     </module>
     <module name="MemberName">
       <property name="format"
-                value="^m_[a-z][a-z0-9][a-zA-Z0-9]*$" />
+                value="^(m_[a-z]([a-zA-Z0-9]*)|value)$" />
       <message key="name.invalidPattern"
                value="Member name ''{0}'' must match pattern ''{1}''." />
     </module>
     <module name="ParameterName">
       <property name="format"
-                value="^[a-z][a-z0-9][a-zA-Z0-9]*$" />
+                value="^[a-z]([a-z0-9][a-zA-Z0-9]*)?$" />
```

Comment:
This is what checkstyle uses to match the Google Style Guide.  It is there to prevent prefixes from being added to parameter names (something that goes against their style guide).